app/src/components/main/analytics/events/profilePageEvents.js

```diff
@@ -14,8 +14,47 @@
  * limitations under the License.
  */
 
+import { getBasicClickEventParameters } from 'components/main/analytics/events/common/ga4Utils';
+import { GA_4_FIELD_LIMIT } from 'components/main/analytics/constants';
+
 export const PROFILE_PAGE = 'profile';
+const basicClickEventParametersProfile = getBasicClickEventParameters(PROFILE_PAGE);
+
 export const PROFILE_PAGE_EVENTS = {
+  // GA4 events
+  CLICK_DELETE_ACCOUNT: {
+    ...basicClickEventParametersProfile,
+    icon_name: 'delete_account',
+  },
+  CONTINUE_BTN_FEEDBACK_MODAL: {
+    ...basicClickEventParametersProfile,
+    element_name: 'continue',
+    modal: 'delete_account',
+  },
+  getDeleteBtnDeleteModalEvent: (checkboxes, otherReason) => ({
+    ...basicClickEventParametersProfile,
+    element_name: 'delete',
+    modal: 'delete_account',
+    status: checkboxes.join('#'),
+    type: otherReason.slice(0, GA_4_FIELD_LIMIT),
```

Comment:
In the document with events, I can not see the `status` and `condition` fields.
Did you discuss these fields with WA?

Reply:
Yes, this is Yuliya's requirement